lib/printer.dart

```diff
@@ -10,8 +10,8 @@ import 'package:source_span/source_span.dart';
 import 'builder.dart';
 import 'src/source_map_span.dart';
 
-const int _LF = 10;
-const int _CR = 13;
+const int lineFeed = 10;
```

Comment:
You made these public, was that intentional?

Reply:
Gah! no! Oops

Reply:
I mean, I wanted to share them, but not make the API public

Reply:
fixed!